helsingborg-stad/app-mitt-helsingborg
# Feat/case summary administrators

## Explain the changes you’ve made

- Implemented new contact persons style in case summary view.
- Added helper function to launch phone app.
- Added helper function to launch email app.

**NOTE**
**The iOS simulator does not support triggering phone call or mail feature. Only works on an actual device.. or Android .** 

## Explain why these changes are made

Implemented latest designs. 

## Explain your solution

Changed the layout of the contact cards and added phone call and mail features as a helper. 

## How to test the changes?

Go to app home screen (Case overview).
Enter case summary screen by clicking on an ongoing or submitted case. 
If the case has any administrators in it's data, it will be showing as contact cards in the view.
Test hitting mail and phone buttons and see if everything works as expected.

## Was this feature tested in the following environments?
- [] Storybook on a iOS device/simulator.
- [] Building the Application on a iOS device/simulator.

## Screenshots (optional)

![Simulator Screen Shot - iPhone 11 - 2020-11-24 at 14 45 17](https://user-images.githubusercontent.com/21363149/100102310-b350f780-2e63-11eb-8744-c2e536dae21f.png)
